resources/js/components/menu.js

```diff
@@ -122,8 +122,9 @@ class MenuComponent extends LitElement {
         <sp-overlay open trigger="trigger@click" placement="bottom" style="position: relative">
           <sp-popover .open="${this.isOpen}">
             <sp-dialog>
-              <h4 slot="heading" class="menu-title">Go to disciple.tools</h4>
               <sp-menu class="right-aligned-menu">
+                <a href="/settings"><h4 slot="heading" class="menu-title">Go to
```

Comment:
@ancy2023 , link this to "/" for now. I asked you to link to /contacts in the meeting, but let's link to the root instead.